src/Network/Network/help/Set-AzNetworkManagerSecurityAdminConfiguration.md

```diff
@@ -0,0 +1,156 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version:
```

Comment:
Please add online version for all new MDs

Reply:
done